Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderContent = () => {
+	return (
+    <div className="headContP indent">
+        <p>Let's learn React by building simple interfaces with components.  Don't try to overthink it, just keep it simple and have fun.  Once you feel comfortable using components you are well on your way to mastering React!</p>
```

Comment:
Same as before, try to use props more to move your data around, this is hard locking you to not be able to ever reuse this component and that is not what you want ultimately. You could simply rename this to be {props.content} then define it in the parent.